### PR TITLE
add example to retrieve lineage graph

### PR DESCRIPTION
**Pull request by andyliucoder**
* Adding an example to retrieve the entire lineage graph. It's been requested by a handful of customers, so this should be enough to get them started.

### CODE REVIEW

File: lineage/lineage_graph_retrieval.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                field: table.node[field]
+                for field in selected_table_fields
+            })
+
+        has_next_page = response.get_tables.page_info.has_next_page
+        if not has_next_page:
+            break
+        current_cursor = response.get_tables.page_info.end_cursor
+
+        time.sleep(sleep_in_seconds)
```
Comment (benitezfede) [inline lineage/lineage_graph_retrieval.py:46]: Why do we need the sleep?

**Review by benitezfede — APPROVED** [review]
LGTM

File: lineage/lineage_graph_retrieval.py (hunk 1 above)
Comment (andyliucoder) [reply lineage/lineage_graph_retrieval.py:46]: We have API limits (I forget exactly, but it limits X number of API requests per minute)

Comment (andyliucoder) [reply lineage/lineage_graph_retrieval.py:46]: so this is a hacky way of getting around it

**Review by bheltzel — APPROVED** [review]
LGTM ;)

**Pull request merged**